pkgs/servers/ankisyncd/default.nix

```diff
@@ -1,68 +1,52 @@
-{ lib
-, fetchFromGitHub
-, python3
-}:
+{ lib, runCommand, fetchFromGitHub, rustPlatform, protobuf }:
```

Comment:
The file should probably be moved aswell.

Reply:
Yes. pname should almost always match the directory name.

I'd do pname and file name as anki-sync-server and the NixOS option as ankisyncd. Then add a package alias `ankisyncd = anki-sync-server;` and also not forget `meta.mainProgram = "ankisyncd";`.

Reply:
Hmm, I'm just really not convinced people would look for anki-sync-server at all -- that's the github repo name alright, but I've always seen it written ankisyncd everywhere else, and I think it is easier to find if package/service directory names match.

I didn't realize pname/directory name should match so I've just reverted pname back to "ankiysncd", and I'd rather just keep it like this for now: the old package's repo was also anki-sync-server packaged as ankisyncd, this just replaces the outdated python implem with the more up to date rust one.

Sorry for the extra round-trips!